presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -527,12 +527,13 @@ private static Table buildTableObject(
 
     private static PrincipalPrivileges buildInitialPrivilegeSet(String tableOwner)
     {
+        PrestoPrincipal grantor = new PrestoPrincipal(USER, tableOwner);
```

Comment:
Please remove the commit "Add grantor to HivePrivilegeInfo", I don't think it's necessary and we can't fill it in accurately.